### PR TITLE
Updating Translations 2012 LKG 02/12/2020

### PR DESCRIPTION
**Pull request by emil-furniss**
Non-forked copy of #1225

### CODE REVIEW

File: components/d2l-quick-eval/lang/de.json [hunk 1]
```diff
@@ -21,30 +21,30 @@
...
    "loadMore" : "Mehr laden",
    "loading" : "Wird geladen",
-   "masterTeacher" : "Lehrer",
+   "masterTeacher" : "Lehrkraft",
    "moreActions" : "Weitere Aktionen",
    "multiCourseQuickEval" : "Kurzbewertung für mehrere Kurse",
    "newAttempts" : "Neue Versuche",
    "newAttemptsDetails" : "{newNum, plural, =0 {{reAttemptNum, plural, =1 {1 neuer Versuch} other {{reAttemptNum} neue Versuche}}} other {{reAttemptNum, plural, =0 {{newNum} neu} =1{{newNum} neu, 1 neuer Versuch} other {{newNum} neu, {reAttemptNum} neue Versuche}}}}",
-   "newPostDetails" : "{numInteractions, plural, =1 {1 Unterhaltung oder Antwort} other {{numInteractions} Unterhaltungen oder Antworten}}",
+   "newPostDetails" : "{numInteractions, plural, =1 {1 Unterhaltung oder Antwort} other {{AnzInterakt} Unterhaltungen oder Antworten}}",
```
Comment (ronan-f) [inline components/d2l-quick-eval/lang/de.json:47]: `numInteractions` looks like a keyword? Should it have been translated?

File: components/d2l-quick-eval/lang/es.json [hunk 2]
```diff
@@ -43,25 +43,25 @@
    "moreActions" : "Más acciones",
    "multiCourseQuickEval" : "Evaluación rápida de múltiples cursos",
    "newAttempts" : "Nuevos intentos",
-   "newAttemptsDetails" : "{newNum, plural, =0 {{reAttemptNum, plural, =1 {1 reintento} other {{reAttemptNum} reintentos}}} other {{reAttemptNum, plural, =0 {{newNum} new} =1{{newNum} new, 1 reintento} other {{newNum} new, {reAttemptNum} reintentos}}}}",
+   "newAttemptsDetails" : "{newNum, plural, =0 {{reAttemptNum, plural, =1 {1 reintento} other {{reAttemptNum} reintentos}}} other {{reAttemptNum, plural, =0 {{newNum} nuevos} =1{{newNum} nuevo, 1 reintento} other {{newNum} nuevos, {reAttemptNum} reintentos}}}}",
```
Comment (ronan-f) [inline components/d2l-quick-eval/lang/es.json:46]: A lot of other `=0` cases were dropped. Should these ones still be here?

**Review by ronan-f — APPROVED** [review]
Left a few comments to be 100% sure but looks correct to me

File: components/d2l-quick-eval/lang/es.json (hunk 2 above)
Comment (mostlyfabulous) [reply components/d2l-quick-eval/lang/es.json:46]: I'll double check it was dropped for the correct langterm

File: components/d2l-quick-eval/lang/de.json (hunk 1 above)
Comment (mostlyfabulous) [reply components/d2l-quick-eval/lang/de.json:47]: Good catch

File: components/d2l-activity-editor/lang/de.js [hunk 3]
```diff
@@ -57,23 +57,25 @@ export default {
 	"editor.noGradeCreatePermission": "Sie haben keine Berechtigung, einen neuen Benotungsgegenstand zu erstellen.", // Reason why a new grade items cannot be created in the choose grades dialog
 	"editor.competencies": "Lernziele", //Text label for the competencies tool integration
 	"editor.manageCompetencies": "Lernziele verwalten", //Button text to launch competencies tool dialog
-	"editor.competenciesCount": "{count, plural, =1 {1 Anhang} other {{Anzahl} Anhänge}}", //Label for number of associated competencies
+	"editor.competenciesCount": "{count, plural, =1 {1 Anhang} other {{count} Anhänge}}", //Label for number of associated competencies
 	"editor.noLearningObjectives": "Keine Lernziele", //text label when there are no associated learning objectives
-	"editor.competenciesCountSummary": "{count, plural, =1 {1 Lernziel} andere {{count} Lernziele}}", // num release condition text
+	"editor.competenciesCountSummary": "{count, plural, =1 {1 Lernziel} other {{count} Lernziele}}", // num release condition text
```
[the remark this answers is not in the document]
Comment (mostlyfabulous) [reply components/d2l-activity-editor/lang/de.js:62]: Keyterm

File: components/d2l-activity-editor/lang/ar.js [hunk 4]
```diff
@@ -63,17 +63,19 @@ export default {
...
+	"editor.specialAccessCount": "يتمتع {count, plural, =1 {مستخدم واحد} other ‏{{count} من المستخدمين}} بإمكانية وصول خاص", // Label for number of special access users
 	"editor.noUsersWithSpecialAccess": "ما من مستخدمين", // text label when there are no users with special access
-	"editor.specialAccessDialogTitle": "إدارة الوصول الخاص", // Dialog title
+	"editor.specialAccessDialogTitle": "إدارة إمكانية الوصول الخاص", // Dialog title
 	"editor.specialAccessHidden": "تم الإخفاء بواسطة الوصول الخاص", // Warning label that the activity is restricted but is being hidden from all users by special access rules
 	"editor.discardChangesTitle": "هل تريد تجاهل التغييرات؟", // Discard Changes User Prompt
 	"editor.discardChangesQuestion": "هل أنت متأكد من أنك تريد تجاهل التغييرات التي أجريتها؟", // Discard Changes User Prompt
 	"editor.yesLabel": "نعم",
 	"editor.noLabel": "لا",
+	"editor.notificationEmailLabel": "بريد إلكتروني للإعلام", // Label for notification email input field
```
[the remark this answers is not in the document]
Comment (mostlyfabulous) [reply components/d2l-activity-editor/lang/ar.js:77]: These are prefixed with editor so they should stay

File: components/d2l-quick-eval/lang/de.json (hunk 1 above)
Comment (mostlyfabulous) [reply components/d2l-quick-eval/lang/de.json]: ✔️

**Review by mostlyfabulous — APPROVED** [review]

**Pull request merged**